cloud_experts_tutorials/cloud-experts-rosa-sts-explained.adoc

```diff
@@ -0,0 +1,150 @@
+
+[id="different-credential-methods-rosa-iam-users"]
+=== Rosa with IAM Users
+When ROSA was first released, the only credential method was ROSA with IAM Users. This method grants IAM users with an AdministratorAccess policy full access to create the necessary resources in the AWS account that uses ROSA. The cluster can then create and expand its credentials as needed.
+
+[id="different-credential-methods-rosa-sts"]
+=== ROSA with STS
+Red Hat improved ROSA to implement security best practices and introduce a method that utilizes AWS STS. ROSA with STS can grant users limited, short-term access to resources in your AWS account. The STS method uses predefined roles and policies to grant temporary least-privilege permissions to IAM users or authenticated federated users. The credentials typically expire an hour after being requested. Once expired, they are no longer recognized by AWS and no longer have account access from API requests made with them. For more information, see the link:https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html[AWS documentation]. While both ROSA with IAM Users and ROSA with STS are currently enabled, ROSA with STS is the preferred and recommended option.
+
+[id="sts-security"]
+== ROSA with STS security
+Several crucial components make ROSA with STS more secure than ROSA with IAM Users:
+
+* An explicit and limited set of roles and policies that the user creates ahead of time. The user knows every requested permission and every role used.
+* The service cannot do anything outside of those permissions.
+* Whenever the service needs to perform an action, it obtains credentials that expire in one hour or less. This means that there is no need to rotate or revoke credentials. Additionally, credential expiration reduces the risks of credentials leaking and being reused.
+
+[id="sts-explained"]
+== AWS STS explained
+ROSA uses AWS STS to grant least-privilege permissions with short-term security credentials to specific and segregated IAM roles. The credentials are associated with IAM roles specific to each component and cluster that makes AWS API calls. This method better aligns with principles of least-privilege and secure practices in cloud service resource management. The ROSA command-line interface (CLI) tool manages the STS roles and policies that are assigned for unique tasks and takes action upon AWS resources as part of OpenShift functionality.
+
+STS roles and policies must be created for each ROSA cluster. To make this easier, the installation tools provide all the commands and files needed to create the roles as policies as well as an option to allow the CLI to automatically create the roles and policies. See xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-sts-creating-cluster-customizations_rosa-sts-creating-a-cluster-with-customizations[Creating a ROSA cluster with STS using customizations] for more information about the different `--mode` options.
```

Comment:
Optional: 
Vale recommends using "and" instead of "as well as". I think either works.